tech/languages/ocaml/ocaml-installation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: OCaml
+subsection: ocaml
+section: tech-languages
+order: 1
+version: 1.0
+description: An open source multi-paradigm programming language and toolchain. 
+---
+
+# What is OCaml?
+
+OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
+
+## OCaml installation
+
+To install the OCaml tools, type on a terminal:
+
+```console
+$ sudo dnf install ocaml
+```
+
+The `ocaml` the REPL (Read-Evaluate-Print-Loop) tool of the language and `ocamlfind` the library manager, `ocamlc` the bytecode compiler and `ocamlopt` the native-code compiler binaries will become available on the system. 
+
+# OPAM 
+
+OPAM (**O**Caml **Pa**ckage **M**anager)  is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow. 
+
+To install OPAM in your Fedora just:
+```console
+$ sudo dnf install opam
+```
+
+To use OPAM to install and manage your OCaml instalation just:
+```bash
```

Comment:
This should be `console` since all this is to be run in a console, right?
````suggestion
```console
````